### ISSUE TITLE
`MAGIC_NUMBER` false positive when `Long` constants are used together with `kotlin.time.Duration` extensions

### ISSUE THREAD

**Issue by 0x6675636b796f75676974687562** [saveourtool/diktat#1934]
## Describe the bug

The following constant declarations trigger the `MAGIC_NUMBER` false positive:

```kotlin
import kotlin.time.Duration.Companion.minutes
import kotlin.time.Duration.Companion.seconds

private val fooTimeout = 10L.seconds

private val barTimeout = 10L.minutes
```

## Environment information
* diktat version: **2.0.0**
* build tool: Maven
* how is diktat run: plugin
* kotlin version: 1.7.22
* operating system: Any

**Comment by rubenquadros** [comment]
How would you solve this? Just check for `.seconds`, `.minutes` and `.hours` from the `nodeText`?

**Comment by orchestr7** [comment]
> How would you solve this? Just check for `.seconds`, `.minutes` and `.hours` from the `nodeText`?

Yes, need to check that the syntax of the constant is the following: 

`val a = (some constant).(any method)` . And if the expression really fits this pattern, then not to raise a warning.

**Comment by wolf-revo** [comment]
@orchestr7 Do you know, that range will also be treated as magic number in same conditions? 

```kotlin
import kotlin.time.Duration.Companion.seconds

val something1 = 1L.seconds // Not a magic number
val something10 = 10.seconds // Magic number
val something10long = 10L.seconds // Magic number
val someRange = 4..11 // Magic number
```

Can't figure out yet, I'm not familiar with this project, but looks like this test should fail, but it passes: `com.saveourtool.diktat.ruleset.chapter3.MagicNumberRuleWarnTest#check ignore ranges `

**Comment by wolf-revo** [comment]
@orchestr7 @0x6675636b796f75676974687562 Kind reminder. Do you think that behavior when range is also treated as magic number at the same conditions is correct?

**Comment by orchestr7** [comment]
> [@orchestr7](https://github.com/orchestr7) [@0x6675636b796f75676974687562](https://github.com/0x6675636b796f75676974687562) Kind reminder. Do you think that behavior when range is also treated as magic number at the same conditions is correct?

Actually I think so, yes, but may be it can be configured by the config flag
